### PR TITLE
Adding Ensemble Model Histogram-based Gradient Boosting Classifier

### PR DESCRIPTION
**Pull request by aolopez**
Adding SKLearn Ensemble Model Histogram-based Gradient Boosting Classifier: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html

### CODE REVIEW

**Review by eyadgaran — APPROVED** [review]
lgtm
